apps/styleguide/src/layout/MolgenisMenu.vue

```diff
@@ -95,14 +100,26 @@ export default {
       //defaut: go home
       return firstItem ? firstItem.href : "../";
     },
+    hasSessionRole() {
```

Comment:
for future, we might want to move these  is.... methods to a more global library as it seems you want use them in other places? For now this is aok.